lua/wire/client/e2descriptions.lua

```diff
@@ -310,8 +310,8 @@ E2Helper.Descriptions["boxCenterW(e:)"] = "Same as using E:toWorld(E:boxCenter()
 E2Helper.Descriptions["aabbMin(e:)"] = "Returns the entity's (min) axis-aligned bounding box"
 E2Helper.Descriptions["aabbMax(e:)"] = "Returns the entity's (max) axis-aligned bounding box"
 E2Helper.Descriptions["aabbSize(e:)"] = "Returns the entity's axis-aligned bounding box size"
-E2Helper.Descriptions["keyClk()"] = "Returns 1 if the E2 was triggered by owner pressing/releasing key"
-E2Helper.Descriptions["keyClk(e)"] = "Returns 1 if the E2 was triggered by player pressing/releasing key"
+E2Helper.Descriptions["keyClk()"] = "Returns the player that pressed/released the key if the E2 was triggered by runOnKey"
+E2Helper.Descriptions["keyClk(e)"] = "Returns 1 or -1 if the E2 was triggered by the player pressing or releasing key"
```

Comment:
Maybe correct this. Something like "Returns 1 if a player pressed or -1 if released a key."